docs/README.md

```diff
@@ -92,41 +77,18 @@ The following table displays the tested Kubernetes and Helm versions.
-| 1.3.0                 | 9.3.4                                     | 1.11.1  | 2.10.1                              | 1.4.0  | 2.11.2         | 1.1.4             | -                        |                        |
-| 1.2.0 - 1.2.3         | 8.13.8                                    | 1.11.1  | 2.8.14                              | 1.1.8  | 2.11.1         | -                 |                          |                        |
-| 1.1.0                 | 8.13.8                                    | 1.8.1   | 2.8.14                              | 1.1.8  | 2.11.1         | -                 |                          |                        |
-| 1.0.0                 | 8.2.0                                     | 1.8.1   | 2.8.1                               | 1.1.6  | 2.7.0          | -                 |                          |                        |
-| 0.17.0 - 0.17.4       | 8.2.0                                     | 1.6.3   | 2.8.1                               | 1.1.0  | 2.7.0          | -                 |                          |                        |
-| 0.14.0 - 0.16.0       | 8.2.0                                     | 1.6.3   | 2.8.1                               | 1.1.1  | 2.7.0          | -                 |                          |                        |
-| 0.13.0                | 8.2.0                                     | 1.6.3   | 2.8.1                               | 1.0.11 | 2.7.0          | -                 |                          |                        |
-| 0.12.0                | 8.2.0                                     | 1.6.3   | 2.8.1                               | 1.0.9  | -              | -                 |                          |                        |
-| 0.9.0 - 0.11.0        | 6.2.1                                     | 1.6.3   | 2.4.4                               | 1.0.8  | -              | -                 |                          |                        |
-| 0.6.0 - 0.8.0         | 6.2.1                                     | 1.6.3   | 2.4.4                               | 1.0.5  | -              | -                 |                          |                        |
+The following table displays the currently used software versions for our Helm chart.
+
+| Name                                      | Version |
+|-------------------------------------------|---------|
+| OpenTelemetry Collector                   | 0.66.0  |
+| OpenTelemetry Operator                    | 0.18.3  |
+| kube-prometheus-stack/Prometheus Operator | 40.5.0  |
+| Falco                                     | 2.4.2   |
+| Telegraf Operator                         | 1.3.10  |
+| Tailing Sidecar Operator                  | 0.5.5   |
+| FluentD                                   | 1.14.6  |
+| Fluent Bit                                | 1.6.10  |
```

Comment:
Perfect case for automation/ci check :star_struck:

Reply:
It is indeed.